src/applications/claims-status/containers/DocumentRequestPage.jsx

```diff
@@ -49,7 +53,10 @@ class DocumentRequestPage extends React.Component {
   componentDidMount() {
     this.props.resetUploads();
     if (this.props.trackedItem) {
-      setDocumentTitle(`Request for ${this.props.trackedItem.displayName}`);
+      const pageTitle = setDocumentRequestPageTitle(
+        this.props.trackedItem.displayName,
```

Comment:
Could this ever be undefined? Is it happen, then the page title would be "Request for undefined"

Reply:
To my knowledge there should always be a display name so im unaware of a reason why it would be undefined or null. I could add a check though and if it is null or undefined then we just use Document Request?